pyvista/plotting/plotting.py

```diff
@@ -3072,6 +3072,8 @@ def _save_image(image, filename, return_img=None):
         if isinstance(filename, str):
             if isinstance(pyvista.FIGURE_PATH, str) and not os.path.isabs(filename):
                 filename = os.path.join(pyvista.FIGURE_PATH, filename)
+            if not any([filename.endswith(ext) for ext in [".png", ".jpeg"]]):
```

Comment:
I don't think any others would be needed here?

Reply:
I thinks that’s good.